sa/database_test.go

```diff
@@ -73,3 +73,26 @@ func TestStrictness(t *testing.T) {
 		t.Fatalf("Got wrong type of error: %s", err)
 	}
 }
+
+func TestSafeSelectOne(t *testing.T) {
```

Comment:
Do you think there should be a test case that shows `safeSelectOne` without an unsafe character in `q` calls the underlying `dbMap.SelectOne` as expected?